MAINTAINERS.yml

```diff
@@ -483,6 +483,7 @@ Documentation:
     labels:
         - "area: ADC"
 
+# See also "Intel Platforms (Xtensa)"
```

Comment:
There is not relation here, there are audio drivers in drivers/audio that are not related to intel platforms, it just happens that the first drivers in audio were for an Intel platform.